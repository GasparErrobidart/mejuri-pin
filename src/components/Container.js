import styled from 'styled-components'

const Container = styled.div`
  padding: 0 ${({gap}) => gap || 20}px;
  box-sizing: border-box;
  width:100%;
`;

export default Container;
