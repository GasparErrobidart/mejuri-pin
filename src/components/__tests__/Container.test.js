import '@testing-library/jest-dom'
import 'jest-styled-components'

import React         from 'react';
import { cleanup }   from '@testing-library/react'
import renderer      from 'react-test-renderer'

import Container     from '../Container'


afterEach(cleanup)

test('Renders "Container" component', () => {
  const testText        = "Container Test Text"
  const component       = renderer.create(<Container>{testText}</Container>);
  const tree            = component.toJSON()
  expect(tree).toMatchSnapshot();
});
