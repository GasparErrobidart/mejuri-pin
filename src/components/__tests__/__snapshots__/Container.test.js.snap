// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "Container" component 1`] = `
.c0 {
  padding: 0 20px;
  box-sizing: border-box;
  width: 100%;
}

<div
  className="c0"
>
  Container Test Text
</div>
`;
